c3po/job/utils/facebook.py

```diff
@@ -49,7 +49,7 @@ def refresh_access_token():
         dotenv.get_key(dotenvfile, "FB_LONG_ACCESS_TOKEN")
         warns = filter(lambda i: issubclass(i.category, UserWarning), warns)
         if warns:
-            request_url = FB_URL + "oauth/access_token"
+            request_url = FB_BASE_URL + "oauth/access_token"
```

Comment:
Shouldn't this remain `FB_URL` ? 🤔

Reply:
I thought so too. But I was running into errors when I ran the script.
Checked the documentation: https://developers.facebook.com/docs/facebook-login/access-tokens#generating-an-app-access-token and found out it is supposed to just be `https://graph.facebook.com/oauth/access_token` for refreshing tokens.

Reply:
Ah ok :+1: